docs/traffic-policy/concepts/cel-interpolation.mdx

```diff
@@ -0,0 +1,90 @@
+# CEL Interpolation
+
+**CEL Interpolation** allows you to write dynamic configurations for traffic policy actions. It enables expressions to be embedded within string fields of a configuration object, making the traffic policy system more flexible and powerful.
+
+When an action in a traffic policy requires a configuration object, some fields allow CEL expressions to be interpolated directly into strings. This means that you can define dynamic values that are computed based on traffic, connection, or previously executed action variables.
+
+For example, when using the `add-headers` action, the `headers` field is a key-value map, where the header values can contain CEL expressions. This allows you to use real-time traffic data, such as the client's IP address, to populate header values dynamically.
```

Comment:
should we add an example policy after this paragraph?

Reply:
I have a WIP for this. This will change after.